feat: add background

NunonuN opened the pull request:
Add background scenario and update some elements of foreground scenario.

Background:

 - Add left mountain;
 - Add right mountain;
 - Add city skyline;

Sky:

 - Change position of star behind tree;

Foreground:

 - Add hear to dog;
 - Change tail of dog;
 - Change size of telescope;
 - Change position and size of hill.

NunonuN committed:
feat: add backgroung

Add background scenario and update some elements of foreground scenario.

Background:

 - Add left mountain;

 - Add right mountain;

 - Add city skyline;

Sky:

 - Change position of star behind tree;

Foreground:

 - Add hear to dog;

 - Change tail of dog;

 - Change size of telescope;

 - Change position and size of hill.

juancarlosfarah commented on src/components/svgs/MountainLeft.js:
```diff
@@ -0,0 +1,31 @@
+      scaleX={scaleX}
+      scaleY={scaleY}
+    />
+    <Path x={x} y={y} data="" fill={fill} scaleX={scaleX} scaleY={scaleY} />
```
@NunonuN, this path is empty, do we need it?

juancarlosfarah commented on src/components/svgs/Hill.js:
```diff
@@ -7,83 +7,76 @@ const Hill = ({ x, y, scaleX, scaleY, fill }) => (
     <Path
       x={x}
       y={y}
-      data="M867.393 759.915c-.446-.928-6.432 1.295-15.009 1.295s-20.175.38-20.654-.613c-.479-.994 7.1-10.752 11.72-15.854 10.81-11.931 14.45-25.528 10.03-37.465-1.569-4.235-2.91-9.065-2.98-10.735-.167-3.956-4.13-11.83-14.216-28.25-9.801-15.957-19.291-37.919-23.321-53.972-2.868-11.42-3.105-11.705-10.288-12.375-4.165-.388-6.972-1.66-6.474-2.935.483-1.237 1.668-2.386 2.633-2.552.966-.167 2.546-.54  ...
```
@NunonuN, this path is empty, do we need it?

juancarlosfarah commented on src/components/svgs/MountainRight.js:
```diff
@@ -0,0 +1,31 @@
+      scaleX={scaleX}
+      scaleY={scaleY}
+    />
+    <Path x={x} y={y} data="" fill={fill} scaleX={scaleX} scaleY={scaleY} />
```
@NunonuN, this path is empty, do we need it?

juancarlosfarah commented on src/components/svgs/City.js:
```diff
@@ -0,0 +1,32 @@
+      scaleX={scaleX}
+      scaleY={scaleY}
+    />
+    <Path x={x} y={y} data="" fill={fill} scaleX={scaleX} scaleY={scaleY} />
```
This path and the one below are empty. Can we remove them?

juancarlosfarah reviewed (changes requested):
@NunonuN, it's looking good in general, but there are some `Path` tags in the SVGs that are empty, so I think that we can remove them.

NunonuN commented:
Yes, we can remove all those empty paths. In fact, I already did it. I left them yesterday by mistake. Sorry about that.
I sent you a message in Slack (need your help with **git**).

NunonuN committed:
Remove `Fragment` from SVGs

Commented out and replaced first line of code in following files inside
src/components/svgs/:

- City.js
- Dog.js
- MountainLeft.js
- MountainRight.js
- TreeLeft.js
- TreeRigh.js

Merge branch 'master' into 26/add-background

NunonuN committed:
Remove `Fragment` from SVGs

Commented out and updated first line of code of the following files
inside the src/components/svgs/ directory:

- City.js
- Dog.js
- MountainLeft.js
- MountainRight.js
- TreeLeft.js
- TreeRight.js

juancarlosfarah commented on src/components/svgs/City.js:
```diff
@@ -1,20 +1,17 @@
-import React, { Fragment } from 'react';
+// import React, { Fragment } from 'react';
```
Can you remove this if no longer needed?

juancarlosfarah commented on src/components/svgs/Dog.js:
```diff
@@ -1,18 +1,17 @@
-import React, { Fragment } from 'react';
+// import React, { Fragment } from 'react';
```
Can you remove this if no longer needed?

juancarlosfarah commented on src/components/svgs/Hill.js:
```diff
@@ -76,7 +76,6 @@ const Hill = ({ x, y, scaleX, scaleY, fill }) => (
       scaleY={scaleY}
     />
-    <Path x={x} y={y} data="" fill={fill} scaleX={scaleX} scaleY={scaleY} />
   </Fragment>
```
Do you not want to remove `<Fragment>` here?

juancarlosfarah commented on src/components/svgs/MountainLeft.js:
```diff
@@ -1,19 +1,17 @@
-import React, { Fragment } from 'react';
+// import React, { Fragment } from 'react';
```
Can you remove this if no longer needed?

juancarlosfarah commented on src/components/svgs/MountainRight.js:
```diff
@@ -1,19 +1,17 @@
-import React, { Fragment } from 'react';
+// import React, { Fragment } from 'react';
```
Can you remove this if no longer needed?

juancarlosfarah commented on src/components/svgs/TreeLeft.js:
```diff
@@ -0,0 +1,29 @@
+// import React, { Fragment } from 'react';
```
Can you remove this if no longer needed?

juancarlosfarah commented on src/components/svgs/TreeRight.js:
```diff
@@ -0,0 +1,29 @@
+// import React, { Fragment } from 'react';
```
Can you remove this if no longer needed?

juancarlosfarah reviewed (changes requested):
Hi @NunonuN, this all looks good! Just some very minor comments. Also, could you add a screenshot to the conversation? That way I can see how it looks without building your branch from scratch. Thanks!

NunonuN committed:
Add background

Merge branch 'master' into 26/add-background
